rest-api-spec/src/main/resources/rest-api-spec/api/indices.rollover.json

```diff
@@ -0,0 +1,30 @@
+{
+  "indices.rollover": {
+    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/indices-rollover-index.html",
+    "methods": ["POST"],
+    "url": {
+      "path": "/{index}/_rollover",
```

Comment:
I am still torn on if we should force the user into our naming pattern. Today we require the user to start with `${some_name}-${some_number}` and we increment that number on each rollover. I think we should make it optional for the user to specify the next indices name maybe via `/{index}/_rollover/{new_index}` and  then skip the pattern enforcement.  @clintongormley what do you think?